src/kixi/datastore/web_server.clj

```diff
@@ -290,32 +299,54 @@
    metrics
    {:id :schema-id-entry
     :methods
-    {:get {:produces "application/transit+json"
+    {:get {:produces "application/json"
            :response
            (fn [ctx]
              (let [id (get-in ctx [:parameters :path :id])]
                (ss/fetch-with schemastore {::ss/id id})))}}}))
 
-(defn add-ns-to-keywords
+(defn add-ns-to-keys
   ([ns m]
    (letfn [(process [n]
              (if (= (type n) clojure.lang.MapEntry)
                (clojure.lang.MapEntry. (keyword (namespace ns) (name (first n))) (second n))
                n))]
      (walk/prewalk process m))))
 
+(defn map-every-nth [f coll n]
+  (map-indexed #(if (zero? (mod %1 n)) (f %2) %2) coll))
+
+(defn keywordize-values
+  [m]
+  (let [keys-values-to-keywordize [::ss/name]
+        m' (reduce
+            (fn [a k]
```

Comment:
personal comment: meh for one-letter parameters :) I agree it's not always easy to name them, but more cognitive effort for the reader